V1/DTOs/Wallet/CreateWalletRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace CoviIDApiCore.V1.DTOs.Wallet
+{
+    public class CreateWalletRequest
+    {
+        public string Photo { get; set; }
+        public string MobileNumber { get; set; }
+        public string MobileNumberReference { get; set; }
```

Comment:
Submitting the photo in this call is interesting. I can see why we'd include it here, because it doesn't feel like it's part of "WalletDetails".

On the other hand, do we want to be storing photos for users who haven't verified their OTP yet?